src/main/java/com/google/cloud/spanner/pgadapter/utils/Metrics.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.cloud.spanner.pgadapter.utils;
+
+import com.google.api.core.InternalApi;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.Meter;
+import java.util.Arrays;
+import java.util.List;
+
+@InternalApi
+public class Metrics {
+  static final String INSTRUMENTATION_SCOPE = "cloud.google.com/java";
+  static final String SPANNER_CLIENT_LIB_LATENCY = "spanner/pgadapter/client_lib_latencies";
+  static final String SPANNER_CLIENT_LIB_LATENCY_DESCRIPTION =
+      "Latency when the client library receives a call and returns a response";
+  static final String PGADAPTER_LATENCY = "spanner/pgadapter/roundtrip_latencies";
+  static final String PGADAPTER_LATENCY_DESCRIPTION =
+      "Latency when the JDBC makes a call and gets a response, measured in PGAdapter";
```

Comment:
Note that the client that connects to PGAdapter can be any PostgreSQL client, not just JDBC. So we should not talk about JDBC as being 'the client'. My suggestion would be:

```suggestion
      "Latency between PGAdapter receiving a statement from the client and PGAdapter returning the last row of the response to the client";
```

Reply:
Updated.